ambuda/templates/proofing/pages/edit.html

```diff
@@ -78,7 +83,8 @@
 <script src="https://cdn.jsdelivr.net/npm/openseadragon@3.1/build/openseadragon/openseadragon.min.js"></script>
 {% set image_url = url_for("site.page_image", project_slug=project.slug, page_slug=cur.slug) %}
 <script type="text/javascript">
-const IMAGE_URL = "{{ image_url }}";
+  const IMAGE_URL = "{{ image_url }}";
 </script>
 <script defer src="/static/js/proofreading.js?v=3"></script>
-{% endblock %}
+<script type="module" src="/static/js/proofreading/prosemirror_bundle.js"></script>
```

Comment:
For now, let's use unpkg.com to serve this file? That way we don't have to deal with checking in the bundle to source control. Later, when we add a bundling system, we can then use `npm install` to get the dependencies.

Reply:
This is a bundle generated from all the other files in this PR (e.g. `prosemirror.ts`, `schema.ts` etc) that were written afresh, rather than an existing bundle already available somewhere. It is the output of:

```
browserify -p tsify ambuda/static/js/proofreading/prosemirror.ts --outfile ambuda/static/js/proofreading/prosemirror_bundle.js
```

(It just occurred to me the names may be a bit confusing; suggestions welcome :-) )

Reply:
One convention is to put generated assets under `static/gen`

Reply:
I see, though, even if we put generated assets here, I would suggest not checking the bundle file into version control. Instead, we can create a build script (maybe similar to `make tailwind_watcher`) that runs browserify to create the bundle. This way it will be easier to handle merge conflicts, etc.

Reply:
Good idea, done.